falcon/request.py

```diff
@@ -1160,6 +1162,8 @@ class RequestOptions(object):
             request's content type is
             *application/x-www-form-urlencoded* (default ``False``). In
             this case, the request's content stream will be left at EOF.
+        auto_parse_query_string_lists: Set to ``False`` in order to disable
```

Comment:
Perhaps it would be helpful here to provide a short example of when this option would be useful.